### PR TITLE
Remove XLNet Squad on v2-8 since it OOMs immediately

### PR DESCRIPTION
**Pull request by allenwang28**

### CODE REVIEW

File: tests/tensorflow/nightly/xlnet-squad.libsonnet [hunk 1]
```diff
@@ -66,7 +65,7 @@ local tpus = import "templates/tpus.libsonnet";
   local v2_8 = {
```
Comment (will-cromar) [inline tests/tensorflow/nightly/xlnet-squad.libsonnet:65]: Should you remove this mixin if there are no working v2-8 configs?

Comment (allenwang28) [reply tests/tensorflow/nightly/xlnet-squad.libsonnet]: Oops, yes done. Also ran the gen script since I forgot.

File: k8s/us-central1/gen/tf-nightly-xlnet-squad-conv-v3-8.yaml [hunk 2]
```diff
@@ -53,7 +53,6 @@
             - "--test_feature_path=$(XLNET_SQUAD_DIR)/spiece.model.slen-512.qlen-64.eval.features.pkl"
             - "--predict_file=$(XLNET_SQUAD_DIR)/dev-v2.0.json"
             - "--predict_dir=$(MODEL_DIR)"
-            - "--train_batch_size=48"
```
Comment (will-cromar) [inline k8s/us-central1/gen/tf-nightly-xlnet-squad-conv-v3-8.yaml:56]: Is this change intentional?

**Review by will-cromar — APPROVED** [review]

File: k8s/us-central1/gen/tf-nightly-xlnet-squad-conv-v3-8.yaml (hunk 2 above)
Comment (allenwang28) [reply k8s/us-central1/gen/tf-nightly-xlnet-squad-conv-v3-8.yaml:56]: Yes, thanks for checking! I had --train_batch_size set in the model command but I prefer it with the accelerator mixin.
